content/pt-br/test/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
No need to copy the files under the test/* directory. Those are there just for testing of the web site framework itself.